server/server/views/recover.py

```diff
@@ -0,0 +1,35 @@
+import transaction
+from passlib.totp import generate_secret
+from pyramid_mailer.mailer import Mailer
+from pyramid_mailer.message import Message
+from passlib.hash import pbkdf2_sha256
+from server.models.user_status import UserStatus
+from pyramid.view import view_config
+
+from ..models.user import User
+
+@view_config(route_name='recover_password', renderer='json')
+def recover_send_mail(request):
+    jsn = request.json_body
+    user = request.dbsession.query(User).filter_by(email=jsn['email']).one_or_none()
+    if user.email is not None:
+        token = request.dbsession.query(User).filter_by(email=jsn['email']).one_or_none()
+        mailer = request.mailer
+        message = Message(subject="hello world",
+                          sender="eventmerv@gmail.com",
```

Comment:
It's a good candidate to go to settings. You may change e-mail in future and will have to update sources, which is not good. To reach settings you can use `request.registry.settings`. ([additional info](https://docs.pylonsproject.org/projects/pyramid/en/1.9-branch/narr/environment.html#adding-a-custom-setting))